generators/typescript/utils/core-utilities/fetcher/src/form-data-utils/__test__/formDataWrapper.test.ts

```diff
@@ -0,0 +1,127 @@
+import { Readable } from "stream"; // Only needed for Node implementations
+import { Node18FormData, WebFormData } from "../FormDataWrapper"; // Adjust path as needed
+
+describe("CrossPlatformFormData", () => {
```

Comment:
can you run seed to make sure this unit test will show up in all the snapshots

Reply:
I ran seed, I think these tests do not get copied over. I will add it!